src/control/PlanarArmController.h

```diff
@@ -42,7 +47,8 @@ template <unsigned int N> class PlanarArmController {
 	void set_setpoint(const navtypes::Vectord<N>& targetJointPos) {
 		Eigen::Vector2d newSetPoint = kin.jointPosToEEPos(targetJointPos);
 		std::lock_guard<std::mutex> lock(mutex);
-		setpoint = newSetPoint;
+		double radius = kin.getSegLens().sum() * safetyFactor;
```

Comment:
This line is duplicated in both usages of `normalizeEEWithinRadius()`, can we just put it in that function?